Yona/Yona/AppDelegate.swift

```diff
@@ -12,6 +12,8 @@ import UIKit
 class AppDelegate: UIResponder, UIApplicationDelegate {
     
     var window: UIWindow?
+    var storyboardName: UIStoryboard?
+    var viewControllers: UIViewController?
     
```

Comment:
App delegate should not keep track of the view controllers and storyboard. Why do we need that?